rfcs/text/0001-rfc-process.md

```diff
@@ -138,35 +220,25 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
 - When all issues are addressed and any required Science reviews are complete:
   - Summarize the review discussion for the **Approvers** in the top-level summary comment in the RFC pull request
-  - Add a minimum one week _last call_ for the completion of the Oversight review to the top-level summary comment in the RFC pull request.
+  
+  **REDACTION HERE**
+  ~~- Add a minimum one week _last call_ for the completion of the Oversight review to the top-level summary comment in the RFC pull request.~~
 
-    **EXAMPLE** 
+    ~~**EXAMPLE**~~ 
 
     ~~**April 1**: Last call for community review~~
 
-    **April 22**: Last call for oversight review
-
-  - Replace "rfc-community-review" with "rfc-oversight-review"
-
-  **NOTE**: *Oversight review is limited to **Approvers**. Further community reviews during this period may be disregarded by the **Author(s)**.*
+    ~~**April 22**: Last call for oversight review~~
 
-- For software RFCs, request an Oversight review of the RFC on the HumanCellAtlas **#tech-architecture** slack channel. Include a link to the RFC pull request and the last call deadline
-
-  Add the RFC as an agenda item to the next *DCP Architecture* meeting
-
-- For governance RFCs, request an Oversight review of the RFC on the HumanCellAtlas **#dcp-project-mgmt** slack channel. Include a link to the RFC pull request and the last call deadline:
-
-    **EXAMPLE** 
-  
-     ***@channel**: Call for oversight review of the RFC process - https://github.com/HumanCellAtlas/dcp-community/pull/27 - Last call is April 22.*
+  ~~- Replace "rfc-community-review" with "rfc-oversight-review"~~
 
-  Add the RFC as an agenda item to the next *DCP PM* meeting
+  ~~**NOTE**: *Oversight review is limited to **Approvers**. Further community reviews during this period may be disregarded by the **Author(s)**.*~~
 
 ### Approving RFCs
```

Comment:
I feel like there's a bit of transition text missing between the Shepherding and Approving. In the original recipe:

> When all issues are addressed and any required Science reviews are complete:

The shepherd would:


> For software RFCs, request an Oversight review of the RFC on the HumanCellAtlas #tech-architecture slack channel. Include a link to the RFC pull request and the last call deadline

> Add the RFC as an agenda item to the next DCP Architecture meeting

but now the Author performs similar actions when Proposing the RFC?

Reply:
I modified it such that Shepherd's are now in charge of making sure the RFC makes it to the next Tech Arch or PM meeting for software and governance RFCs respectively. That task has been removed from the Author task list under Proposing the RFC.

Reply:
Will close this for now, but please do re-open if this wasn't made more clear.